### PR TITLE
fix: useCookieState  documentation colon escape

### PR DESCRIPTION
**Pull request by xiaotiandada**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

before
image

after
image
image

### CODE REVIEW

File: packages/hooks/src/useCookieState/demo/demo3.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * title: Use the option property to configure Cookie
- * desc: Available options: defaultValue、expires、path、domain、secure、sameSite etc.
+ * desc: Available options&#58; defaultValue、expires、path、domain、secure、sameSite etc.
```
Comment (hchlq) [inline packages/hooks/src/useCookieState/demo/demo3.tsx]: 除了这个方式之外，还有其他的方法么，比如有什么特殊符号不解析后面这个 `:`

Comment (xiaotiandada) [reply packages/hooks/src/useCookieState/demo/demo3.tsx]: > 除了这个方式之外，还有其他的方法么，比如有什么特殊符号不解析后面这个 `:`

` * desc: Available options:&nbsp;defaultValue、expires、path、domain、secure、sameSite etc.`

这样是不是比较好理解，保留 `:`

更好的体验可能需要修改插件的渲染正则 🤔

Comment (hchlq) [reply packages/hooks/src/useCookieState/demo/demo3.tsx]: 这个可以～

**Review by hchlq — APPROVED** [review]

File: packages/hooks/src/useCookieState/demo/demo3.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  * title: Use the option property to configure Cookie
- * desc: Available options: defaultValue、expires、path、domain、secure、sameSite etc.
+ * desc: Available options:&nbsp;defaultValue、expires、path、domain、secure、sameSite etc.
```
Comment (hchlq) [inline packages/hooks/src/useCookieState/demo/demo3.tsx]: 改成这样更好点～

` desc: 'Available options: defaultValue、expires、path、domain、secure、sameSite etc.'`

**Review by hchlq — APPROVED** [review]

File: packages/hooks/src/useCookieState/demo/demo3.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /**
  * title: Use the option property to configure Cookie
- * desc: Available options: defaultValue、expires、path、domain、secure、sameSite etc.
+ * desc: 'Available options: defaultValue、expires、path、domain、secure、sameSite etc.'
```
Comment (crazylxr) [inline packages/hooks/src/useCookieState/demo/demo3.tsx:3]: 现在这样改造之后描述还是没展示出来呢

**Review by crazylxr — APPROVED** [review]

**Pull request merged**